client/jetpack-onboarding/main.jsx

```diff
@@ -30,43 +31,54 @@ class JetpackOnboardingMain extends React.PureComponent {
 	// TODO: Add lifecycle methods to redirect if no siteId
 
 	render() {
-		const { siteId, siteSlug, stepName, steps } = this.props;
-
+		const { recordTracksEventForJpoSite, siteId, siteSlug, stepName, steps } = this.props;
 		return (
 			<Main className="jetpack-onboarding">
 				<Wizard
 					basePath="/jetpack/onboarding"
 					baseSuffix={ siteSlug }
 					components={ COMPONENTS }
-					siteId={ siteId /* Passed down to individual steps */ }
-					steps={ steps }
-					stepName={ stepName }
 					hideNavigation={ stepName === STEPS.SUMMARY }
+					recordTracksEventForJpoSite={ recordTracksEventForJpoSite }
```

Comment:
I wonder if we could figure a shorter name...maybe `recordJpoEvent` or something...